.github/workflows/stale-pr.yml

```diff
@@ -0,0 +1,17 @@
+name: "Manage stale pull requests"
```

Comment:
The other workflows are named as "Activate", "CI", "Assign". Not important, but something more compact like "Stale PRs" may look more consistent, and make things easier to find in the workflows list.